.github/workflows/linux-jdks-with-current-maven-ci.yml

```diff
@@ -30,7 +30,7 @@ jobs:
   openjdk:
     strategy:
       matrix:
-        platform: [3.2.5-jdk-8, 3.5.4-jdk-8-slim, 3.6.3-openjdk-11-slim, 3.8.7-eclipse-temurin-17-alpine, 3.9.0-eclipse-temurin-19-alpine, 3.9.1-eclipse-temurin-20-alpine]
+        platform: [3.6.3-openjdk-11-slim, 3.8.7-eclipse-temurin-17-alpine, 3.9.0-eclipse-temurin-19-alpine, 3.9.1-eclipse-temurin-20-alpine]
```

Comment:
> With parent update - we still support Java 8.

I can't believe after these changes. It is not tested with Java 8 any more.

Reply:
Good catch. My intention was to remove old Maven, not Java 8. I'll fix it